pom.xml

```diff
@@ -868,6 +869,46 @@
             </dependency>
 
             <!-- 3rd party -->
+            <dependency>
+                <groupId>com.aliyun.datalake</groupId>
+                <artifactId>metastore-client-common</artifactId>
+                <version>${dep.aliyun-dlf.version}</version>
+                <exclusions>
```

Comment:
Should this be
```xml
<exclusions>
    <exclusion>
        <groupId>*</groupId>
        <artifactId>*</artifactId>
    </exclusion>
</exclusions>
```
Why are all of these excluded? If they're not used, can you fix the upstream artifact to only include the required dependencies?